sw/device/silicon_creator/lib/sigverify/BUILD

```diff
@@ -110,6 +110,10 @@ opentitan_test(
     srcs = ["mod_exp_ibex_functest.c"],
     cw310 = cw310_params(
         timeout = "long",
+        tags = [
+            # The wycheproof suite has a lot of tests; run it only on the nightly CI.
+            "nightly",
```

Comment:
I wonder whether it would make sense to change this so that these tests were tagged "whycheproof" and then the "only run whycheproof tests in nightly regressions" decision would only need to be done and documented in one place.

I'm not a Bazel expert though: this might be really silly!

(Same question below for the known answer tests)